elements/urth-viz-chart/urth-viz-chart-common-behavior.html

```diff
@@ -442,7 +442,21 @@
                         }
 
                         // Use selected class on DOM to maintain selected state
-                        this.$$('.nvd3 .hover').classList.toggle('selected');
+                        var hovered = this.$$('.nvd3 .hover');
+
+                        // IE does not support classList on SVG elements
+                        if (hovered.classList) {
+                            hovered.classList.toggle('selected');
+                        } else if (hovered.getAttribute) {
+                            var classList = hovered.getAttribute('class').split(' ');
```

Comment:
Could use d3 instead, except it doesn't have a convenient toggle method.  6 vs 1/2 doz?

```
var hovered = d3.select(this.$$('.nvd3 .hover'));
hovered.classed('selected', !hovered.classed('selected'));
```